README.md

```diff
@@ -1,3 +1,39 @@
 # Ada Developers Academy Jump Start
 
-This repository contains the materials presented during the Ada Developers Academy jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+This repository contains the materials presented during the [Ada Developers Academy](adadevelopersacademy.org) jump start program. This program is intended for accepted Ada Developers Academy students to give them a jump start on learning materials and concepts that will be utilized throughout our year-long program.
+
```

Comment:
I think we should position this as jump starting anyone's career into programming, whether they come to Ada or do something else.